packages/react/src/css-map/index.ts

```diff
@@ -16,22 +15,22 @@ import type { Pseudos } from './pseudos';
  * };
  * ```
  */
-type CssProps = Readonly<Properties<string | number>>;
+type CSSProperties = Readonly<Properties<string | number>>;
 
-type AllPseudos = { [key in Pseudos]?: CssProps & AllPseudos };
+type AllPseudos = { [key in Pseudos]?: CSSProperties & AllPseudos };
 
 // The `screen and (max-width: 768px)` part of `@media screen and (max-width: 768px)`.
 // Ideally we would do type checking to forbid this from containing the `@media` part,
```

Comment:
What's this comment touching on? Do template literal types solve this?

Reply:
This is an old comment from @liamqma - do u have context?

Reply:
To write a media query in CSS map, we gonna do 
```
const foo = cssMap({
   "@media": {
       "screen and (max-width: 768px)": {
           color: 'red'
       }
   }
})
```
Note `@media screen and (max-width: 768px)` is segmented into two parts. This idea is borrowed from vanilla-extract https://vanilla-extract.style/documentation/styling/#media-queries.

The comment says ideally we should check the second segment does not include "@media" as it's redundant.